yazi-core/src/folder/files.rs

```diff
@@ -76,8 +82,12 @@ impl Files {
 		async fn go(entities: &[DirEntry]) -> Vec<File> {
 			let mut files = Vec::with_capacity(entities.len() / 3 + 1);
 			for entry in entities {
-				if let Ok(meta) = entry.metadata().await {
-					files.push(File::from_meta(Url::from(entry.path()), meta).await);
+				match entry.metadata().await {
+					Ok(meta) => files.push(File::from_meta(Url::from(entry.path()), meta).await),
+					Err(_) => {
+						let file_type = entry.file_type().await.ok();
```

Comment:
I'm wondering if this is necessary - if a file's metadata can't be got, can we just get its filetype? I've never seen a broken file, so I doubt its necessity.